packages/koa-shopify-webhooks/src/test/register.test.ts

```diff
@@ -96,4 +96,42 @@ describe('registerWebhook', () => {
     const result = await registerWebhook(webhook);
     expect(result.success).toBe(false);
   });
+
+  it('sends an eventbridge registration GraphQL query for an eventbridge webhook registration', async () => {
+    fetchMock.mock('*', successResponse);
+    const webhook: Options = {
+      address: 'myapp.com/webhooks',
+      topic: 'PRODUCTS_CREATE',
+      accessToken: 'some token',
+      shop: 'shop1.myshopify.io',
+      apiVersion: ApiVersion.Unstable,
```

Comment:
Might be worth using the api version that it was released in 2020-04. That's the version we're telling people in the docs so having a test here lining up is better than unstable.

Reply:
✅